crates/bevy_window/src/window.rs

```diff
@@ -266,38 +277,43 @@ impl Window {
         self.resolution.scale_factor()
     }
 
-    /// The cursor position in this window
+    /// The cursor position in this window in logical pixels.
+    ///
+    /// Returns `None` if the cursor is outside the window area.
     #[inline]
     pub fn cursor_position(&self) -> Option<Vec2> {
         self.internal
             .physical_cursor_position
             .map(|position| (position / self.scale_factor()).as_vec2())
     }
 
-    /// The physical cursor position in this window
+    /// The cursor position in this window in physical pixels.
+    ///
+    /// Returns `None` if the cursor is outside the window area.
     #[inline]
     pub fn physical_cursor_position(&self) -> Option<Vec2> {
         self.internal
             .physical_cursor_position
             .map(|position| position.as_vec2())
     }
 
-    /// Set the cursor position in this window
+    /// Set the cursor position in this window in logical pixels.
```

Comment:
Is there an explanation of logical/physical we can link here?

Reply:
Probably to `WindowResolution`'s physicial_width/height, same for all the other cursor method and also `scale_factor`.

Reply:
Maybe resolved by https://github.com/bevyengine/bevy/pull/8858/commits/a36128c631ab0677c3cfd13e09ff2026af4b5315 ?

I added " See [WindowResolution] for an explanation about logical/physical sizes." when needed